packages/core/src/openapi/openapi-request-builder.ts

```diff
@@ -38,6 +44,19 @@ export class OpenApiRequestBuilder<ResponseT = any> {
     return this;
   }
 
+  /**
+   * Add custom request configs to the request.
```

Comment:
This also applies to OData: As a user I would like to have a little bit more context of what this is going to be used for. Will it just be added to the request as is?